### PR TITLE
Azkaban 3.0 documentation

### PR DESCRIPTION
**Pull request by logiclord**
* Moved older latest (2.6) documentation to 2.6 directory.
* Created new documentation for latest i.e. 3.0.0 release.
* Verified formatting by hosting locally using github-pages

### CODE REVIEW

File: _includes/docs/latest/getting-started.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <h1 id="getting-started">Getting Started</h1>
 </div>
 
-<p class="lead">In version 2.5 we provide two modes: the stand alone "solo-server" mode and the heavier weight two server mode. The following describes the differences between the two modes.</p>
+<p class="lead">In version 3.0 we provide three modes: the stand alone "solo-server" mode, the heavier weight two server mode and distributed multiple-executor mode. The following describes the differences between the two modes.</p>
```
Comment (johnyu0520) [inline _includes/docs/latest/getting-started.html]: do we still want to mention the 2 server mode?

File: _includes/docs/latest/azkaban-webserver.html [hunk 2]
```diff
@@ -55,6 +55,60 @@ <h2 id="azkaban-webserver">Azkaban Web Server Configurations</h2>
...
+	</thead>
+	<tbody>
+		<tr>
+			<td>azkaban.use.multiple.executors</td>
+			<td>Should azkaban run in multi-executor mode. Required for multiple executor mode.</td>
+			<td>false</td>
+		</tr>
+		<tr>
+			<td>azkaban.executorselector.filters</td>
+			<td>Hard filters to be used while dispatching. To be choosen from StaticRemaining, FlowSize, MinimumFreeMemory and CpuStatus.</td>
```
Comment (johnyu0520) [inline _includes/docs/latest/azkaban-webserver.html]: Do we want to mention
1. it's possible to choose multiple
2. what is the delimiter if want to choose multiple

File: _includes/docs/latest/azkaban-webserver.html [hunk 3]
```diff
@@ -55,6 +55,60 @@ <h2 id="azkaban-webserver">Azkaban Web Server Configurations</h2>
...
+			<td> </td>
+		</tr>
+		<tr>
+			<td>azkaban.executorselector.comparator.{ComparatorName}</td>
```
Comment (johnyu0520) [inline _includes/docs/latest/azkaban-webserver.html]: Do we want to mention
1. it's possible to choose multiple
2. what is the delimiter if want to choose multiple
3. Does ordering of the comparator matter? (I remember we are suppose to change the weighting somewhere?)

File: _includes/docs/latest/executor-setup.html [hunk 4]
```diff
@@ -73,12 +73,26 @@ <h2 id="executor-setup">Set up Azkaban Executor Server</h2>
...
+<h4>Single Executor Mode</h4>
 <p>By default the <code>executor.port</code> is set to <code>12321</code>. The AzkabanWebServer will have to point to this port as well.</p>
 <p>This is done by setting the following property in AzkabanWebServer's <code>azkaban.properties</code>.</p>
 <pre class="code">
 executor.port=12321
 </pre>
 
+<h4>Multiple Executor Mode</h4>
+<p>We need to enable multiple executor mode in webserver configuration if we want to run in multiple executor mode. Please verify that you have following properties in azkaban.properties.
+azkaban.use.multiple.executors and azkaban.executorselector.comparator.* are required properties. Please note that <code></code> is not honoured in multiple exeu</p>
```
Comment (johnyu0520) [inline _includes/docs/latest/executor-setup.html]: exeu: typo? (or maybe it renders ok)

Comment (logiclord) [inline _includes/docs/latest/executor-setup.html]: Good catch. Yes, that's a typo.
Fixed.

File: _includes/docs/latest/azkaban-webserver.html (hunk 3 above)
Comment (logiclord) [inline _includes/docs/latest/azkaban-webserver.html]: Multiple will have separate property. As discussed offline,  I have added an example and made specification more clearer by specifying that it is an integer weight and what can be the value for Memory {ComparatorName}.

File: _includes/docs/latest/getting-started.html (hunk 1 above)
Comment (logiclord) [inline _includes/docs/latest/getting-started.html]: As discussed offline, we are keeping all 3 modes for the time being as some user might want to use 2 serve mode.

File: _includes/docs/latest/azkaban-webserver.html (hunk 2 above)
Comment (logiclord) [inline _includes/docs/latest/azkaban-webserver.html]: Good catch. Added statements to specify both the points.
Thanks.

**Comment by logiclord** [comment]
Incorporated feedback.

**Comment by johnyu0520** [comment]
LGTM

File: _includes/docs/latest/getting-started.html (hunk 1 above)
Comment (tigerpeng2001) [inline _includes/docs/latest/getting-started.html]: I think it would be better to have links point to old versions.

**Pull request merged**